src/styles/variables/_z-index.scss

```diff
@@ -1,3 +1,4 @@
 /* Used to set the item at the highest z-index and then directly below */
```

Comment:
While looking at other components, noticed there are other "fugitive" z-index variables of 999 inhabiting other classes out there. Just a word of caution for us all as we add more instances of this to be conscious of how these will interact with each other in relation to their [stacking context](https://developer.mozilla.org/en-US/docs/Web/CSS/CSS_Positioning/Understanding_z_index/The_stacking_context). This is more of a comment/ word of caution going forward to avoid adverse stacking scenarios.

Reply:
@theilman7 I think this is a good call, we should move these to variables.